doc/data/messages/d/deprecated-class/pylintrc

```diff
@@ -0,0 +1,2 @@
+[main]
+py-version = 3.3
```

Comment:
This one is implicit as we never used py-version before python 3.5 and we're supporting from python3.7 upward.

Reply:
The example I found for deprecated-class was said to be for Python 3.3